server/sonar-server/src/main/java/org/sonar/server/platform/ServerIdLoader.java

```diff
@@ -45,11 +45,11 @@ public ServerIdLoader(Settings settings, ServerIdGenerator idGenerator) {
       return Optional.absent();
     }
 
-    String organisation = settings.getString(CoreProperties.ORGANISATION);
+    String organization = settings.getString(CoreProperties.ORGANIZATION);
```

Comment:
I don't think that we could do that. Licenses are handled by a plugin, I don't know it but if it's using ORGANISATION it will fail.